File: P2N_Pet_BackEnd/P2N_Pet_API/P2N_Pet_API/Database/PetShopModels/Breed.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace P2N_Pet_API.Database.PetShopModels
{
    public partial class Breed
    {
        public Breed()
        {
            InverseBreedNavigation = new HashSet<Breed>();
            Pets = new HashSet<Pet>();
        }

        public ulong Id { get; set; }
        public string Name { get; set; }
        public ulong? Breedid { get; set; }
        public int? Status { get; set; }
        public ulong? Createuser { get; set; }
        public DateTime? Createdate { get; set; }
        public ulong? Updateuser { get; set; }
        public DateTime? Updatedate { get; set; }

        public virtual Breed BreedNavigation { get; set; }
        public virtual ICollection<Breed> InverseBreedNavigation { get; set; }
        public virtual ICollection<Pet> Pets { get; set; }
    }
}

File: P2N_Pet_FrontEnd/P2N-Pet/projects/admin/src/app/models/breed.ts
export class Breed{
    public Id: number;
    public Name: string;
    public BreedIdName: string;
    public StatusText: string;
    public CreateUserName: string;
    public CreateDate: string;
    public UpdateUserName: string;
    public UpdateDate: string;
}

export class BreedCondition {
    Name: string;
    BreedId: string;
    Status: string;
  
    constructor() {
      this.Name = '';
      this.BreedId = '0';
      this.Status = '0';
    }
  }

  export class BreedDefaultSelection {
    Id: number;
    Name: string;
  }